packages/skill-lesson/path-to-purchase/pricing.tsx

```diff
@@ -164,7 +167,7 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
     product
   const {subscriber, loadingSubscriber} = useConvertkit()
   const router = useRouter()
-  const [autoApplyPPP, setAutoApplyPPP] = React.useState<boolean>(true)
+  const [autoApplyPPP, setAutoApplyPPP] = React.useState<boolean>(false)
```

Comment:
@jbranchaud turning this off made the PPP upgrade work as expected. i remember this previous discussion and the logic is bonkers 😭